repose-aggregator/src/docs/asciidoc/developer/docker.adoc

```diff
@@ -45,3 +46,49 @@ Since this method will not automatically tag the image, you will need to tag it
 docker tag IMAGE_ID local/repose:v{project-version}-ubuntu
 ----
 
+== Logging inside the Repose Docker Container
+The published images adhere to the expectations of 12Factor logging.
```

Comment:
I might include a snippet about `docker logs` here to show the standard way of accessing logs.

Reply:
Updated.